examples/README.md

```diff
@@ -1,30 +1,52 @@
 # Examples
 
-The `ballerinax/hubspot.crm.object.feedback` connector provides practical examples illustrating usage in various scenarios.
+The `HubSpot CRM Feedback` connector provides practical examples illustrating usage in various scenarios.
 
-[//]: # (TODO: Add examples)
-1. 
-2. 
+[Feedback Reviewing](https://github.com/module-ballerinax-hubspot.crm.object.feedback/tree/main/examples/) - This example demonstrates the usage of the HubSpot CRM Feedback connector to read a page of feedback submissions, read an object identified by `{feedbackSubmissionId}`, read a batch of feedback submissions by internal ID, or unique property values, and search feedback submissions.
+
+> **Note**: The feedback submissions endpoints are currently read only. Feedback submissions cannot be submitted or edited through the API. You can only create properties in the [feedback surveys tool within HubSpot](https://knowledge.hubspot.com/customer-feedback/create-a-custom-survey?_gl=1*1h5ce0o*_ga*MzIyODQzMTUyLjE3MzYzNjE3OTU.*_ga_LXTM6CQ0XK*MTczNjQwNjg4NS4zLjEuMTczNjQwNjg5OS40Ni4wLjA.&_ga=2.97072604.213367396.1736361795-322843152.1736361795#survey), and the properties cannot be edited after creation.
 
 ## Prerequisites
 
-[//]: # (TODO: Add prerequisites)
+- **Ballerina:** Download and install Ballerina from [here](https://ballerina.io/downloads/).
+- **HubSpot developer account:** Create a HubSpot developer account and create an app to obtain the necessary credentials. Refer to the [Setup Guide](../ballerina/Package.md) for instructions.
+- **`hubspot.crm.object.feedback` module:** Import the `ballerinax/hubspot.crm.object.feedback` module into your Ballerina project and configure it with the obtained credentials. Refer to the [Config.toml.template](./feedback_review/Config.toml.template) file for creating the `Config.toml` file.
+
+```ballerina
+import ballerinax/hubspot.crm.obj.feedback as feedback;
+
+configurable string & readonly clientId = ?;
+configurable string & readonly clientSecret = ?;
+configurable string & readonly refreshToken = ?;
+
+public function main() returns error?{
+    feedback:ConnectionConfig config = {
+        auth: {
+            clientId,
+            clientSecret,
+            refreshToken,
+            credentialBearer: oauth2:POST_BODY_BEARER
+        }
+    };
+
+    final feedback:Client baseClient = check new feedback:Client(config, serviceUrl);
```

Comment:
Move the client initialization outside of the `main` function. Check formatting as well.